stations/abc.py

```diff
@@ -12,28 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pytz
-import requests
-from datetime import datetime
+from urllib.request import urlopen
+
+from bs4 import BeautifulSoup
 
 def abc():
     """Custom news fetcher for ABC News Australia briefing"""
-    # Format template with (hour, day, month)
-    url_temp = ('https://abcmedia.akamaized.net/news/audio/news-briefings/'
-                'top-stories/{}{}/NAUs_{}00flash_{}{}_nola.mp3')
-    now = pytz.utc.localize(datetime.utcnow())
-    syd_tz = pytz.timezone('Australia/Sydney')
-    syd_dt = now.astimezone(syd_tz)
-    hour = syd_dt.strftime('%H')
-    day = syd_dt.strftime('%d')
-    month = syd_dt.strftime('%m')
-    year = syd_dt.strftime('%Y')
-    url = url_temp.format(year, month, hour, day, month)
-
-    # If this hours news is unavailable try the hour before
-    response = requests.get(url)
-    if response.status_code != 200:
-        hour = str(int(hour) - 1)
-        url = url_temp.format(year, month, hour, day, month)
-
-    return url
+    domain = "https://www.abc.net.au"
```

Comment:
Why is this custom fetcher in its own file while the other is at the top of the `__init__.py` file?  Also, functions do something so in general they should be named with a verb as the first word.

Reply:
This was the first, plan was to come back for the rest. Have moved them all and renamed to a consistent `get_station_url()`